src/ape/managers/converters.py

```diff
@@ -140,6 +141,27 @@ def convert(self, value: Union[List, Tuple]) -> Union[List, Tuple]:
 list_tuple_converter = ListTupleConverter(None, None, None)  # type: ignore
 
 
+class TimestampConverter(ConverterAPI):
```

Comment:
Can we make one that also accepts `datetime.datetime` objects?

Reply:
Instead of a string? I don't see why not

Reply:
I didn't add this to this issue though, I think we should add a separate issue for this, what do you think?

Reply:
Yes a separate issue is just fine.
This way though, we don't have to worry about trying to cover every possible string format because the user can just parse their own `datetime` objects so I think it is helpful.

Reply:
completely agree